docs/deployment.md

```diff
@@ -1,61 +1,83 @@
+All emails should be added to AWS SES in the account/region where the tooling environment resides.
 
-Assuming that your target AWS profile is the default one set, execute the following from /infra_tooling_account/ :
+![AWS SES Configuration](/docs/images/ses-identities.png "AWS SES Configuration")
 
-```cdk deploy --all --context stage-name=<your_stage_name>```
+## Preparing SALMON settings
 
-Optionally you can add an AWS profile identifier via --profile command argument.
+You need to prepare configuration files for the solution and place them in the **/config/** folder.  
 
-## Monitored Environment Setup
+For more details on Configuration, see [Documentation](/docs/configuration.md).
 
-- Monitored Environment is any combination of AWS Account and Region which needs to be monitored.
-- Even if one of AWS Account and Region has Tooling stack configured, it still needs Monitored Environment deployment if you need to monitor services residing within this Account and Region
-- stage-name parameter needs to be defined for deployment as an identifier of deployment stage (usually dev/test/qa/uat/prod, but can be any string identifier)
+You can also refer to sample settings files in **/config/sample_settings/**.  
 
-Assuming that your target AWS profile is the default one set, execute the following from /infra_monitored_account/ :
+## CDK Deploy: Tooling Environment
 
+Once all prerequisites are met and the configuration is ready, it's time to deploy all artifacts to AWS.
+Deployment starts with the Tooling Environment:
+
+- browse into **infra_tooling_account** folder
+- run the following command  
 ```cdk deploy --all --context stage-name=<your_stage_name>```
 
-Optionally you can add an AWS profile identifier via --profile command argument.
\ No newline at end of file
+Optionally, you can add an AWS profile identifier via the '--profile' command argument.
+
+*`stage-name` is a parameter which identifies the deployment stage (usually dev/test/qa/uat/prod, but can be any string identifier of your choice).*
+
+What happens during *`cdk deploy`* of tooling environment:
+- Configuration files are validated (e.g. if delivery methods referred to in *recipients.json* are aligned with those defined in *general.json*)
+- Configuration files are copied to S3 bucket (where all SALMON components will read the configuration from).
+- AWS resources are created. For a list of resources, please refer to [Architecture document](/docs/architecture.md).
+
+*Note: If you change the settings (e.g. add a new recipient), you will need to run the aforementioned "cdk deploy" command to validate new config files and upload them onto S3.*
+
+*Note: If you choose to use optional Grafana component, make sure you to have a VPC with a public subnet and a security group prepared. The security group should allow access to port 3000 from any IP address.*
```

Comment:
would it be useful to mention how to deploy-destroy only Grafana stack ?
cdk deploy cf-salmon-InfraToolingGrafanaStack-<your_stage_name>  --context stage-name=<your_stage_name>
cdk destroy cf-salmon-InfraToolingGrafanaStack-<your_stage_name> --context stage-name=<your_stage_name>

Reply:
or probably we can mention this case in FAQ

Reply:
That's a good point. I'm planning to get rid of this nuance in ver 1.1 (using Nested Stacks, probably).
There will be a follow-up task